indico/migrations/versions/20181023_1051_55893713f6b7_add_room_name_format.py

```diff
@@ -0,0 +1,26 @@
+"""Add room_name_format
+
+Revision ID: 55893713f6b7
+Revises: 93985a8c11ed
+Create Date: 2018-10-23 10:51:55.999263
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '55893713f6b7'
+down_revision = '93985a8c11ed'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('locations',
+                  sa.Column('room_name_format', sa.String(), nullable=False, server_default=u'%1$s/%2$s-%3$s'),
```

Comment:
- no need for the unicode prefix
- remove the `server_default` (`alter_column` to set it to None) after adding the column to be in sync with the model

Reply:
👍